sources/metrics/tracking pixels/TrackingPixelsProperties.swift

```diff
@@ -0,0 +1,84 @@
+//  Copyright 2019, Oath Inc.
+//  Licensed under the terms of the MIT License. See LICENSE.md file in project root for terms.
+
+import Foundation
+import PlayerCore
+
+enum TrackingPixels {
+    struct Properties {
+        let vrm: VRM
+        struct VRM {
+            let vrmResponseStatus: ResponseStatus
+            enum ResponseStatus {
+                case noResponse
+                case response(transactionID: String?)
+                
+                init(response: VRMResponse?) {
+                    switch response {
+                    case .none:
+                        self = .noResponse
+                    case .some(let value):
+                        self = .response(transactionID: value.transactionId)
+                    }
+                }
+            }
+            let scheduledItems: [VRMCore.Item: Set<ScheduledVRMItems.Candidate>]
+            let timeoutBarrier: Double
+            let completedItems: Set<VRMCore.Item>
+            let timeoutedItems: Set<VRMCore.Item>
+            let erroredItems: Set<VRMCore.Item>
+            let responseTime: [VRMCore.Item: VRMItemResponseTime.TimeRange]
+            let timeoutStatus: VRMProcessingTimeout
+        }
+        let trackingInfo: TrackingInfo
```

Comment:
Can we list out all properties at the top and put declaration to the bottom - currently it is hard to read.